### ISSUE TITLE
bazel is not able to download the certificates and causing error

### ISSUE THREAD

**Issue by systemlogic** [bazelbuild/rules_scala#532]
https://github.com/bazelbuild/rules_scala/blob/dec85a95d13f403f360cb51829f1804d0518165a/scala/scala.bzl#L297

Solutions: Certificate can be ignored or fall back the protocol to http.

Can someone bring the change or simply suggest the solution?
-harish
IBMer

```
ERROR: /root/.cache/bazel/_bazel_root/8c069df52082beee3c95ca17836fb8e2/external/io_bazel_rules_scala/scala/scala.bzl:295:3: no such package '@scala//': Error downloading [https://downloads.lightbend.com/scala/2.11.11/scala-2.11.11.tgz] to /root/.cache/bazel/_bazel_root/8c069df52082beee3c95ca17836fb8e2/external/scala/scala-2.11.11.tgz: sun.security.validator.ValidatorException: PKIX path validation failed: java.security.cert.CertPathValidatorException: timestamp check failed and referenced by '//external:io_bazel_rules_scala/dependency/scala/scala_reflect'
```

**Comment by johnynek** [comment]
Solution: copy that function in a file in your repo and replace the url with one of your choosing. A local url that you control would be a good idea so your builds don’t depend on anything else. We run a local nexus that mirrors all our external deps.

**Comment by Code0x58** [comment]
I emailed [Lightbend](https://github.com/lightbend) to let them know their cert has expired, and asked if they have this hosted on GitHub or if they could.

**Comment by ittaiz** [comment]
fixed by @systemlogic in #533 
thanks!
